crates/matrix-sdk-ui/src/authentication/mod.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg(feature = "experimental-oidc")]
```

Comment:
So this seemed to make sense, but now running `cargo test` doesn't run the tests in here. Does CI run the tests with all features enabled?

Reply:
> I guess the FFI crate will, in the future?

Oh I forgot to add that change in this commit 🙈